app/instance-initializers/bugsnag.js

```diff
@@ -24,6 +23,8 @@ export default {
       Ember.onerror = function(error) {
         Bugsnag.context = getContext(router);
         const metaData = getMetaData(error, owner);
+        const user = getUser(owner) || {};
```

Comment:
@binhums I'm happy to test this, but think it warrants a larger refactor, specifically splitting this `onerror` callback out so it can be tested in isolation. Were you looking for that in this PR? Or did you have another approach in mind?